pkg/node/nodes.go

```diff
@@ -45,23 +47,28 @@ func NodesRunningNmstate(cli client.Reader, nodeSelector map[string]string) ([]c
 func MaxUnavailableNodeCount(cli client.Reader, policy *nmstatev1.NodeNetworkConfigurationPolicy) (int, error) {
 	enactmentsTotal, _, err := enactment.CountByPolicy(cli, policy)
 	if err != nil {
-		return 0, err
+		return MIN_MAXUNAVAILABLE, err
 	}
 	intOrPercent := intstr.FromString(DEFAULT_MAXUNAVAILABLE)
 	if policy.Spec.MaxUnavailable != nil {
 		intOrPercent = *policy.Spec.MaxUnavailable
 	}
 	maxUnavailable, err := ScaledMaxUnavailableNodeCount(enactmentsTotal, intOrPercent)
-	return maxUnavailable, nil
+	return maxUnavailable, err
```

Comment:
We should add some linting to the project... I will try to put in place golangci-lint.

We can do directly
```golang
return ScaledMaxUnavailableNodeCount(enactmentsTotal, intOrPercent)
```

Reply:
Ah, of course...

Reply:
Done